### DISCUSSION TITLE
Whisper transcribes my entire audio file into single timestamp, please help

### DISCUSSION THREAD

**Discussion by lukasons** [openai/whisper#1863] [Q&A] [answered]
Hi everyone, I am new with whisper and I need help. I am using whisper locally. When i transcribe my audio file, entire text gets under one single timestamp:
[00:00.000 --> 00:35.080] And entire text is here...

coman I use in CMD is whisper file_name.mp3 or mp4 it detects my language OK.

I have also tried adding --word_timestamps True --max_words_per_line 8 but same thing

**Comment by ryanheise** [comment, marked as answer]
What you see on the screen is just a progress indicator, but the final output is stored in files such as `file_name.srt` (with 8 words per line) and `file_name.json` for raw output including word timestamps.

**Reply by lukasons** [reply]
I feel so stupid, thanks!

**Reply by lukasons** [reply]
For some reason it worked today but later on same thing happens, here is text from .srt file, everything is one big line


00:00:00,000 --> 00:00:18,000
Evo sada moram da je isprobam ovo snimanje zvuka i da vidim kako se pojavljuju titlovi, dalje se odvajaju u posebne redove kada snimanje traje malo duže, dalje ću biti zadvoljen, ja i dalje snimam, nadam sa da je zvuk ok.

**Reply by lukasons** [reply]
OK so you were still right, after restaring and reinstalling everything, works fine!
